dnf-behave-tests/dnf/search.feature

```diff
@@ -35,13 +35,17 @@ Scenario: with installed and availiable newest package doesn't duplicate results
    Then the exit code is 0
    And stdout is
    """
-   ======================== Name & Summary Matched: setup =========================
-   setup.noarch : A set of system configuration and setup files
-   setup.src : A set of system configuration and setup files
+   <REPOSYNC>
+   Matched fields: name (exact), summary
+    setup.noarch: A set of system configuration and setup files
+    setup.src: A set of system configuration and setup files
    """
 
 
-Scenario: with installed and availiable newest package and --showduplicates duplicates results
+# TODO(jkolarik): Do we want this in DNF5? Current package query behavior seems different 
+# and here are two same NEVRAs...
```

Comment:
The redundant NEVRA is for the installed packages (which technically is a different package, although the user cannot see the difference from the output).
I guess current dnf5 behavior is OK until we add a possibility for the user to have also repoid in the output (e.g. something like --queryformat, or some kind of structured output like json).